test_requirements.txt

```diff
@@ -2,7 +2,6 @@
 colour-runner==0.0.4
 coverage==3.7.1
 dj-database-url==0.3.0
-django==1.7.4
```

Comment:
I'm never sure whether it's better to leave this here and let travis re-install it, or to leave it out and expect users to manually install it.